packages/dev/gui/src/2D/controls/image.ts

```diff
@@ -859,10 +859,18 @@ export class Image extends Control {
             return;
         }
 
-        const canvas = this._workingCanvas!;
-        context = canvas.getContext("2d")!;
+        const transform = context.getTransform();
 
-        context.drawImage(this._domImage, sx, sy, sw, sh, tx - this._currentMeasure.left, ty - this._currentMeasure.top, tw, th);
+        const canvas = this._workingCanvas!;
+        const workingCanvasContext = canvas.getContext("2d")!;
+        workingCanvasContext.save();
+        const ttx = tx - this._currentMeasure.left;
+        const tty = ty - this._currentMeasure.top;
+        workingCanvasContext.setTransform(transform.a, transform.b, transform.c, transform.d, (ttx + tw) / 2, (tty + th) / 2);
+        workingCanvasContext.translate(-(ttx + tw) / 2, -(tty + th) / 2);

```

Comment:
Is it not the same as:
```typescript
workingCanvasContext.setTransform(transform.a, transform.b, transform.c, transform.d, 0, 0);
```
?

Reply:
`setTransform` clears the existing matrix, but `transform` pushes a new transformation to the matrix, the intention is to move the center of the context to the center of the object, do the rotation/scaling, then move back, like:

(T2)(T1)p, where p is a point, T1 is the matrix on like 869, and T2 the one on 870.